Sources/Client/WebSocket/WebSocket.swift

```diff
@@ -360,6 +367,23 @@ extension WebSocket: WebSocketProviderDelegate {
         
         return nil
     }
+    
+    private func handleTypingEvent(_ event: Event) {
+        guard let user = event.user, !user.isCurrent else {
+            return
+        }
+        
+        if let timerControl = typingEventTimeoutTimerControls[user] {
+            timerControl.cancel()
+            typingEventTimeoutTimerControls[user] = nil
+        }
+        
+        if case .typingStart = event {
+            typingEventTimeoutTimerControls[user] = Timer.schedule(timeInterval: 30, queue: provider.callbackQueue) { [weak self] in
+                self?.publishEvent(.typingStop(user, event.cid, .typingStop))
```

Comment:
You're going to send `typingStop` event for a user other than current?
In line 372, you `return` if `user.isCurrent` so this `user` is not current

Reply:
Good point. fixed in 7566711

Reply:
@buh What do you mean? I don't follow